modules/alloy/src/main/ts/ephox/alloy/ui/schema/FloatingToolbarButtonSchema.ts

```diff
@@ -42,7 +43,8 @@ const parts: () => PartType.PartTypeAdt[] = Fun.constant([
           aria: {
             mode: 'expanded'
           },
-          toggleOnExecute: false
+          toggleOnExecute: false,
+          onToggled: (_api, state) => detail.onToggled(state)
```

Comment:
In terms of adding a comment, I probably meant something like 

Here, we would put

```
// For FloatingToolbars, we can hook up our `onToggled` handler directly to the Toggling because we don't have to worry about any animations. Unfortunately, for SlidingToolbars, Toggling is more directly hooked into the animation for growing, so to have an event `onToggled` that doesn't care about the animation, we can't just hook into the Toggling config.
```

And then in the SlidingToolbar button Toggling config, either copy the same comment, or just say "See comment in ___" (which might get out-of-date)

Reply:
Added the comments [here](https://github.com/tinymce/tinymce/pull/8238/commits/76b67657d40baa31fd7443df16fb080bb2edd9ff).